tracer/src/Datadog.Tracer.Native/version.h

```diff
@@ -1,3 +1,9 @@
 #pragma once
 
 constexpr auto PROFILER_VERSION = "2.53.0";
+
+#ifdef _WIN32
+constexpr auto TRACER_ASSEMBLY = L"Datadog.Trace, Version=2.53.0.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb";
+#else
+constexpr auto TRACER_ASSEMBLY = u"Datadog.Trace, Version=2.53.0.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb";
+#endif
```

Comment:
cannot you use `WStr` ?
if I'm correct, you would be able to have only one line:
`constexpr auto TRACER_ASSEMBLY = WStr("Datadog.Trace, Version=2.53.0.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb");`

Reply:
Oh does it mean that you will have to update the `Version`, and the `PublicKeyToken` at each new version ? (maybe not the public key token, but the version for sure)

Reply:
I didn't want to use `WSr` to avoid including something in the file. And yes, the only updateable part is the version, so I'd like to construct the assembly string from it, but I haven't been able so far.

Reply:
I see, what you can do
```
#define DD_TRACER_VERSION "2.53.0"
constexpr auto PROFILER_VERSION = DD_TRACER_VERSION;

#ifdef _WIN32
constexpr auto TRACER_ASSEMBLY = L"Datadog.Trace, Version=" DD_PROFILER_VERSION ".0, Culture = neutral, PublicKeyToken = def86d061d0d2eeb";
#else
constexpr auto TRACER_ASSEMBLY = u"Datadog.Trace, Version=" DD_PROFILER_VERSION ".0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb";
#endif

#undef DD_TRACER_VERSION
```
It's not as elegant as I would have liked but it does the job

Reply:
Whichever approach we go with, we'll have to make sure to update the "version updater" Nuke task to account for this 🙂

Reply:
Yep, I put it in the `version.h` file for that particular reason